src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+                  You can see how others add topics (but you can't read them
+                  until the voting step).
+                </Text>
+              </Box>
+              <Box maxWidth={["100%", null, "45%", "20%"]}>
+                <LogInCircle height={32} />
+                <Text as="h3" mt={5} mb={4} textStyle="landingHighlightsTitle">
+                  No login required
+                </Text>
+                <Text textStyle="landingHighlightsText">
+                  A retro is just a url, nothing else is needed. We don't even
+                  ask for your name or email!
+                </Text>
+                <Text textStyle="landingHighlightsText" mt={4}>
+                  And if you want to keep a retro secret, passwords are coming.
+                </Text>
+              </Box>
+            </Flex>
+          </Box>
+        </Section>
+        <Box
+          id="how-it-works"
+          py={[9, null, null, 120, 150]}
+          px={6}
+          fontSize={3}
+          lineHeight={8}
+          overflow="hidden"
+        >
+          <Box width="100ch" maxWidth="100%" margin="auto" position="relative">
+            <Screenshot display={["none", null, null, null, null, "block"]} />
+            <Box
+              width="60ch"
+              maxWidth="100%"
+              margin={["auto", null, null, null, null, 0]}
+            >
+              <Text as="h2" fontSize={[6, null, 7, 8]} fontWeight="bold">
+                How to use Retro tool
+              </Text>
+              <Text fontSize={[4, null, 5]} color="grey" my={4}>
+                The main and only screen is a 4 columns layout where you can add
+                different topics and action items (unless you are on tablet or
+                mobile, in that case the UI will adapt).
+              </Text>
+              <Text fontSize={[4, null, 5]} color="grey" mb={[8, null, 10]}>
+                A retro consists in 4 simple guided steps, you can always see
+                the current step in the breadcrumb and you can advance to the
+                next step with the button placed in the header.
+              </Text>
+
+              <RetroSteps>Brainstorm</RetroSteps>
+              <Text textStyle="landingBase" mt={1} mb={8}>
+                The first screen is where{" "}
+                <strong>you and your team can add topics</strong> like{" "}
+                <em>what went well</em>, <em>what can be improved</em> and{" "}
+                <em>questions</em>, usually this step is done asynchronously. A
+                good idea is to share the retro url some days before the retro
+                meeting.
+              </Text>
+
+              <RetroSteps>Group &amp; vote</RetroSteps>
+              <Text textStyle="landingBase" mt={1} mb={8}>
+                This step is to <strong>group and vote</strong> so you know what
+                are the most important topics for your team. Usually this step
```

Comment:
I think the correct phrasing is `so you know what the most important topics are for your team`, or `so you know what the most important topics for your team are`, depending on whether you want to put more weight on the _team_ aspect or on the _topics_. I'd go with the first option.